app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -1648,6 +1648,7 @@ public boolean onPrepareOptionsMenu(final Menu menu) {
             menu.setGroupVisible(R.id.photos_option_men, false);
             menu.findItem(R.id.all_photos).setVisible(!editMode && !hidden);
             menu.findItem(R.id.search_action).setVisible(!editMode);
+            menu.findItem(R.id.settings).setVisible(false);
```

Comment:
Can you remove it from the menu instead of setting it hidden? Or are there any other trouble?

Reply:
First I thought of the same thing of removing it. But If I remove it, the settings option will not be available in the overflow menu when you open an album.

Reply:
Thanks!